xchange-cexio/src/main/java/org/knowm/xchange/cexio/service/CexIOTradeService.java

```diff
@@ -79,6 +80,9 @@ public boolean cancelOrder(String orderId) throws IOException {
   public boolean cancelOrder(CancelOrderParams orderParams) throws IOException {
     if (orderParams instanceof CancelOrderByIdParams) {
       return cancelOrder(((CancelOrderByIdParams) orderParams).getOrderId());
+    } else if (orderParams instanceof CancelOrderByCurrencyPair) {
+      cancelCexIOOrders(((CancelOrderByCurrencyPair) orderParams).getCurrencyPair());
+      return true;
     } else {
       return false;
```

Comment:
Would be nice to throw an exception here, if the parameter type is wrong.
```suggestion
          throw new IllegalArgumentException(String.format("Unknown parameter type: %s", orderParams.getClass()));
```